ts/modules/export-data.src.ts

```diff
@@ -523,6 +523,25 @@ Highcharts.Chart.prototype.getDataRows = function (
                 dateTimeValueAxisMap: dateTimeValueAxisMap
             };
         },
+        // Create point array depends if xAxis is category
+        // or point.name is defined #13293
+        getPointArray = function (
+            series: Highcharts.Series,
+            xAxis: Highcharts.Axis
+        ): string[] {
+            if (
+                series.data.filter((d: Highcharts.Point): string | undefined => d.name).length &&
+                xAxis && !xAxis.categories &&
+                series.type !== 'gantt'
```

Comment:
Avoid checking `series.type`. Instead use `series.is('series-type')`. 

But why, in the first place, we need to filter out `gantt` type? How about x-range?

Reply:
`Gantt` chart has defined properly the `series.pointArrayMap` - (`start, end`) and doesn't require any additional changes like adding the 'x' to this array.